source/_docs/energy.markdown

```diff
@@ -21,3 +21,17 @@ Home Assistant is an open platform and so home energy management is not restrict
 If you have a sensor that returns instantaneous power readings (W or kW), then to add a sensor that returns energy usage or generation (kWh) refer to [Riemann sum integral integration](/integrations/integration/#energy)
 
 <img src='/images/docs/energy/energy-overview.png' alt='Visual representation of how all different energy forms relate.' style='border: 0;box-shadow: none;'>
+
+## Troubleshooting Missing Entities 
+
+If you have a sensor you are trying to add to the energy dashboard, and it does not appear in the selection list, please check the following points to see why it may be missing:
+
+- The sensor must have the appropriate attributes. Check your entity attributes in `Developer Tools -> States` to confirm the following:
```

Comment:
```suggestion
- The sensor must have the appropriate attributes. Check your entity attributes in **{% my developer_states title="Developer Tools > States" %}** to confirm the following:
```
these [My links](https://my.home-assistant.io/faq/) take you directly to the location in your instance.